modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -4,123 +4,27 @@
-			24-round magazine and is compatible with suppressors."
-	item = /obj/item/gun/ballistic/automatic/c20r/unrestricted
-	cost = 14
-
-/datum/uplink_item/dangerous/shotgun_traitor
-	name = "Bulldog Shotgun"
-	desc = "A fully-loaded semi-automatic drum-fed shotgun. Compatible with all 12g rounds. Designed for close \
-			quarter anti-personnel engagements."
-	item = /obj/item/gun/ballistic/shotgun/bulldog/unrestricted
-	cost = 13
-
-/datum/uplink_item/dangerous/shield_traitor
-	name = "Energy Shield"
-	desc = "An incredibly useful personal shield projector, capable of reflecting energy projectiles and defending \
-			against other attacks. Pair with an Energy Sword for a killer combination."
-	item = /obj/item/shield/energy
-	cost = 5
-
 /datum/uplink_item/dangerous/katana_traitor
 	name = "Katana"
 	desc = "An incredibly sharp sword used by Samurais. Woefully underpowered in D20."
 	item = /obj/item/katana
-	cost = 7
+	cost = 8
+	progression_minimum = 20 MINUTES
 
 /datum/uplink_item/dangerous/oddjob
 	name = "Deadly Bowler Hat"
 	desc = "An incredibly sharp edged bowler hat used by an infamously short operative."
 	item = /obj/item/clothing/head/sus_bowler
 	cost = 20
 
-//STEALTHY WEAPONS
-/datum/uplink_item/stealthy_weapons/cqc_traitor
-	name = "CQC Manual"
-	desc = "A manual that teaches a single user tactical Close-Quarters Combat before self-destructing."
-	item = /obj/item/book/granter/martial/cqc
-	cost = 23
-	surplus = 17
-
-// Removed from the uplink for the time being.
-/*datum/uplink_item/stealthy_weapons/cqcplus
-	name = "CQC+ Manual"
-	desc = "A manual that teaches a single user tactical Close-Quarters Combat and how to deflect projectiles before self-destructing."
-	item = /obj/item/book/granter/martial/cqc/plus
-	cost = 30
-	surplus = 20
-*/
-
-/datum/uplink_item/stealthy_weapons/telescopicbaton
-	name = "Telescopic Baton"
-	desc = "A telescopic baton, exactly like the ones heads are issued. Good for knocking people down briefly."
-	item = /obj/item/melee/baton/telescopic
-	cost = 2
-	surplus = 0
-
 //STEALTHY TOOOLS
-/datum/uplink_item/stealthy_tools/infiltratormask
-	name = "Voice-Muffling Balaclava"
-	desc = "A balaclava that muffles your voice, masking your identity. Also provides flash immunity!"
-	item = /obj/item/clothing/mask/infiltrator
-	cost = 2
-
-/datum/uplink_item/stealthy_tools/deluxe_agent_card
-	name = "Deluxe Agent Identification Card"
-	desc = "Created by Cybersun Industries to be the ultimate for field operations, this upgraded Agent ID \
-	comes with all the fluff of the original, but with an upgraded microchip - allowing for the storage of all \
-	standard Nanotrasen access codes in one conveinent package. Now in glossy olive by default!"
-	item = /obj/item/card/id/advanced/chameleon/black
-	cost = 5 //Since this gives the possibility for All Access, this is a BIGBOY tool. Compared to oldbases' skeleton key, though, you still have to steal it somehow.
```

Comment:
I don't think this is an issue. Up the cost a little, maybe.

Reply:
I'm not convinced letting players buy all access capable cards from the uplink is good, it devalues the shit out of the captain's ID and his spare ID and makes them far less valuable targets